WordPress/UITests/Utils/XCTest+Extensions.swift

```diff
@@ -3,6 +3,9 @@ import XCTest
 
 extension XCTestCase {
 
+    // Require main actor isolation because of the calls to app.terminate() and app.activate()
+    // which require running on the main thread.
+    @MainActor
     public func setUpTestSuite(
```

Comment:
Since `setUpTestSuite` is main actor-isolated, all its callers need to be as well, which is way we have all those `@MainActor` additions in this PR.